auslib/test/test_db.py

```diff
@@ -1311,7 +1316,7 @@ def testValidateConditionsTelemetryRaisesError(self):
 
     @mock.patch("time.time", mock.MagicMock(return_value=200))
     def testInsertWithEnabledCondition(self):
-        what = {"fooid": 11, "foo": "i", "bar": "jjj", "data_version": 1, "when": 909000, "change_type": "insert"}
+        what = {"fooid": 11, "foo": "i", "bar": "jjj", "data_version": 1, "when": 909000, "change_type": "update"}
```

Comment:
Hmm...I'm not sure I understand what you mean by verified here, can you clarify?